### ISSUE TITLE
Publish npm package for basis trancoder

### ISSUE THREAD

**Issue by lilleyse** [BinomialLLC/basis_universal#247]
We recently integrated the basis transcoder in CesiumJS while adding support for `KHR_texture_basisu`. Everything is working great so far.

Now we're starting to pull all our third party libraries from npm and we would be very interested in an npm package for the basis transcoder. There's a package on npm [node-basis-wrapper](https://www.npmjs.com/package/node-basis-wrapper) that isn't really usable for us since it ships with just the basisu windows executable and isn't maintained.

@richgel999 is this something you're be interested in? We would be happy to contribute if any code changes are required.

CC @pjcozzi @ebogo1

Related CesiumJS issues:

* https://github.com/CesiumGS/cesium/issues/9473
* https://github.com/CesiumGS/cesium/pull/9513

**Comment by richgel999** [comment]
Yes, we're very interested in this. I investigated how to do it myself, but I have no experience creating npm packages. Any help would be awesome.

**Comment by pjcozzi** [comment]
@richgel999 very cool.

@ebogo1 @lilleyse let's do it! 🚀

**Comment by notmike101** [comment]
Hey, wondering if there's any updates on this.
